### PR TITLE
Vue tour

### PR DESCRIPTION
**Pull request by tecimovic**
Ongoing PR from `vue-tour` branch, so we have CI and we're ready to merge this in.

### CODE REVIEW

File: src/components/ZclClusterManager.vue [hunk 1]
```diff
@@ -239,6 +255,30 @@ export default {
   components: {
     ZclDomainClusterView,
   },
+  data() {
+    return {
```
Comment (jingteng25742) [inline src/components/ZclClusterManager.vue:259]: we shouldn't be hardcoding the state of the UI within some var.

File: src/components/ZclClusterManager.vue [hunk 2]
```diff
@@ -239,6 +255,30 @@ export default {
...
+      expendedCluster: {
```
Comment (jingteng25742) [inline src/components/ZclClusterManager.vue:260]: expendedCluster - typo for expandedCluster?

File: src/components/ZclCommandManager.vue [hunk 3]
```diff
@@ -50,6 +50,38 @@ limitations under the License.
             >
               {{ validationErrorMessage }}
             </q-tooltip>
+            <!-- <q-icon
```
Comment (jingteng25742) [inline src/components/ZclCommandManager.vue:53]: not related to Vue Tour.
Is this being accidentally added due to merge/rebase?

**Review by jingteng25742 — CHANGES_REQUESTED** [review]
Requesting change after code review @Amirreza-Ghasemkhani 
cc: @tecimovic

File: src/components/ZclCommandManager.vue (hunk 3 above)
Comment (Amirreza-Ghasemkhani) [reply src/components/ZclCommandManager.vue:53]: some of these commented code came from some of rebase/merge I should check the code again I think

**Review by jingteng25742 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**